apps/passport-server/migrations/49_kudosbot_uploaded_proofs.sql

```diff
@@ -0,0 +1,5 @@
+create table kudosbot_uploaded_proofs (
+  id SERIAL PRIMARY KEY,
+  proof VARCHAR NOT NULL,
+  uploadTime TIMESTAMP NOT NULL DEFAULT NOW()
```

Comment:
```suggestion
  createdAt TIMESTAMP NOT NULL DEFAULT NOW()
```

Reply:
I guess one question is - do we need this column at all?

Reply:
this might be useful for rate limiting kudoses -- right now anyone can keep giving out kudoses (in fact they could spam the upload endpoint with the same proof over and over again), so the `createdAt` column would be helpful for implementing rate limiting (alternatively, the rate limiting could be stored in a separate table)

Reply:
sure, rate limit sounds useful enough for the spamming use case